src/core/event-handler.js

```diff
@@ -168,7 +168,10 @@ class EventHandler {
             evt.callback.call(evt.scope, arg1, arg2, arg3, arg4, arg5, arg6, arg7, arg8);
 
             if (evt.once) {
-                const ind = this._callbacks[name].indexOf(evt);
+
+                const existed_callbacks = this._callbacks[name];
+                const ind = existed_callbacks ? existed_callbacks.indexOf(evt) : -1;
```

Comment:
```suggestion
                // check that callback still exists because user may have unsubscribed
                // in the event handler
                const existingCallback = this._callbacks[name];
                const ind = existingCallback ? existingCallback.indexOf(evt) : -1;
```

Reply:
Added a comment and use more standard naming convention.